src/app/application.cpp

```diff
@@ -804,6 +809,18 @@ void Application::applyMemoryWorkingSetLimit()
 #endif
 }
 
+#ifdef Q_OS_WIN
+void Application::adjustThreadPriority() const
+{
+    // Workaround for improving responsiveness of qbt when CPU resources are scarce.
+    // Raise Qt threads to be just one level higher than libtorrent threads. However, this behavior is not
+    // always the same on all OS. Also note that on *nix platforms there is no easy way to achieve it,
+    // so implementation is omitted.
+
+    ::SetThreadPriority(::GetCurrentThread(), THREAD_PRIORITY_ABOVE_NORMAL);
```

Comment:
That is not possible (or very cumbersome, even if done on libtorrent side) on Windows since as said before, child threads won't inherit parent priority. Every child thread would need to set priority explicitly if it needs to be different from the default.

I don't insist on this PR but if you insist on lowering libtorrent threads priority, then this PR will come to an end.

Reply:
> or very cumbersome, even if done on libtorrent side

Is it more difficult than setting thread memory priority, which has already been done in libtorrent?

Reply:
>Is it more difficult than setting thread memory priority, which has already been done in libtorrent?

Yes. As mentioned before, it would need to be applied to each new thread explicitly.
Also libtorrent doesn't apply memory priority to every thread they use. They only apply on a few selected disk threads IIRC.

Reply:
> libtorrent doesn't apply memory priority to every thread they use. They only apply on a few selected disk threads IIRC.

Ok.